cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/templates/etl/batch/BatchETLDBAdapterTest.java

```diff
@@ -103,7 +98,7 @@ public static void setup() throws Exception {
     // deploy etl batch template
     String path = Resources.getResource("org/hsqldb/jdbcDriver.class").getPath();
     File hsqldbJar = new File(URI.create(path.substring(0, path.indexOf('!'))));
-    templateManager = deployApplication(ETLBatchTemplate.class, hsqldbJar);
+    //templateManager = deployApplication(ETLBatchTemplate.class, hsqldbJar);
```

Comment:
wanted to leave it as a reference when doing the db test. it will get removed soon.